org.eclipse.xtext.ide.tests/testlang-src/org/eclipse/xtext/ide/tests/testlanguage/TestLanguageRuntimeModule.xtend

```diff
@@ -23,12 +27,20 @@ class TestLanguageRuntimeModule extends AbstractTestLanguageRuntimeModule {
 		return TestLanguageFormatter;
 	}
 	
-    def Class<? extends ISignatureHelpService> bindSignatureHelpService() {
-        return SignatureHelpServiceImpl;
-    }
-    
-    def Class<? extends IColoringService> bindIColoringService() {
-    	return ColoringServiceImpl;
-    }
+	def Class<? extends ISignatureHelpService> bindSignatureHelpService() {
+		return SignatureHelpServiceImpl;
+	}
+
+	def Class<? extends IColoringService> bindIColoringService() {
```

Comment:
Is this binding still needed. I assume we don't need it.

Reply:
Why not? Although the service is deprecated we did not remove it; hence it makes sense to test it, right?

Reply:
That would imply that we keep the call in the LanguageServerImpl. I would prefer to remove it. Any reason besides testing against that?

Reply:
If we remove the call from the LS, why do we keep the `IColoringService` at all? I mean, what is the benefit of a deprecated interface (and its default implementation) if it is not called from anywhere?

Reply:
Ok. If someone has used it then we still need to call it in LS - fair enough.